docs/source/developer_guide/git_workflow.rst

```diff
@@ -10,14 +10,14 @@ GitHub, clone, and develop on a new branch.
 
 The workflow consists of two main parts:
 
-* **Creating a fork**: This sets up your own copy of the ``sktime`` repository on GitHub.
+* **Creating a fork and cloning the repository**: This sets up your own copy of the ``sktime`` repository on GitHub.
   This needs to be done only once, when you start contributing to ``sktime``.
 * **Developing a feature**: This is the process of developing a new feature, e.g.,
   a bugfix or new estimator.
   This is done every time you want to contribute a new feature.
 
 
-Creating a fork - Initial one time setup
+Creating a fork and cloning the repository
```

Comment:
undlerlines should always be same number of character.

I think it is important to say "one-time setup", it was a common confusion point that new contributors repeated the fork/clone.